modules/document_repository/test/TestPlan.md

```diff
@@ -28,20 +30,11 @@
    [Manual Testing]
 14. Edit a file in the repository: check that “Date Uploaded” date is updated.
     [Manual Testing]
-15. Check that the preference “Receive Document Repository Notifications” works.
-    [Manual Testing]
-16. Check that the performance is OK with large number of files.
-    [Manual Testing]
-17. Check that if you have the 'Receive document repository notification' activated you will receive an email each 
-    time one of the following event occurs:
+15. Visit the My Preferences module and enable notifications for the document_repository.
+    Make sure that you are notified for the follow changes:
        - Addition, deletion or modification of a file (by a user other than yourself)
        - Addition of a category (by a user other than yourself)
-    Also check that the www address contained in the notification email is correct.
     [Manual Testing]
-18. To upload large files, check that the following settings are in php.ini (default: /etc/php/{php_version}/apache2/php.ini):
-      session.gc_maxlifetime = 10800
-      max_input_time = 10800
-      max_execution_time = 10800
-      upload_max_filesize = 1024M
-      post_max_size = 1024M 
+16. Try uploading a large file, e.g. 1 GB. Ensure that an error message occurs
```

Comment:
@johnsaigle too large is relative, this item should read `try uploading a file larger then the max upload size ...`